STATUS.md

```diff
@@ -0,0 +1,489 @@
+    - [ ] Waypoints
+    - [ ] Tasks
+  - [ ] Ground
+    - [ ] Group
+    - [x] Units
+    - [ ] Waypoints
+    - [ ] Tasks
+  - [ ] Plane
+    - [ ] Group
+    - [ ] Units
+    - [ ] Waypoints
+    - [ ] Tasks
+    - [ ] Loadout
+  - [ ] Helicopters
+    - [ ] Group
+    - [ ] Units
+    - [ ] Waypoints
+    - [ ] Tasks
+    - [ ] Loadout
+- [ ] `addStaticObject`
+- [x] `getGroups`
+- [ ] `getStaticObjects`
+- [x] `getAirbases`
+- [x] `getPlayers`
+- [ ] `getServiceProviders`
+- [ ] `addRefPoint`
+- [ ] `getRefPoints`
+- [x] `getMainRefPoint` ("Bullseye")
+- [ ] `getCountryCoalition`
+
+### Coord Singleton
+- [ ] <del>`LLtoLO`</del> (internally, used; but LAT/LNG is used in API definitions)
+- [ ] <del>`LOtoLL`</del> (internally, used; but LAT/LNG is used in API definitions)
+- [ ] <del>`LLtoMGRS`</del> Rather implement using conversion: https://en.wikipedia.org/wiki/Military_Grid_Reference_System
+- [ ] <del>`MGRStoLL`</del> (internally, used; but LAT/LNG is used in API definitions)
+
+### Env Singleton
+The following API's are not planned to be exposed via gRPC; client applications
+should use their independent logging and tracing functions.
+- [ ] <del>`info`</del>
+- [ ] <del>`warning`</del>
+- [ ] <del>`error`</del>
+- [ ] <del>`setErrorMessageBoxEnabled`</del>
+- [ ] <del>`getValueDictByKey`</del>
+
+### Land Singleton
+- [ ] `getHeight`
+- [ ] `getSurfaceHeightWithSeabed`
+- [ ] `getSurfaceType`
+- [ ] `isVisible`
+- [ ] `getIP`
+- [ ] `profile`
+- [ ] `getClosestPointOnRoads`
+- [ ] `findPathOnRoads`
+
+### Mission Commands Singleton
+- [x] `addCommand`
+- [x] `addSubMenu`
+- [x] `removeItem`
+- [x] `addCommandForCoalition`
+- [x] `addSubMenuForCoalition`
+- [x] `removeItemForCoalition`
+- [x] `addCommandForGroup`
+- [x] `addSubMenuForGroup`
+- [x] `removeItemForGroup`
+
+### Net Service
+- [x] `net.send_chat`
+- [x] `net.send_chat_to`
+  - (not implemented `fromId` -- open to use cases)
+- [ ] `recv_chat`
+- [x] `get_player_list`
+  - Implementation compounded with `get_player_info`
+- [ ] `get_my_player_id`
+- [ ] `get_server_id`
+- [x] `get_player_info`
+  - Returned as a detailed response with compounded `get_player_list`
+- [ ] `kick`
+- [ ] `get_stat`
+- [x] <strike>`get_name`</strike> (will not be implemented -- returned part of `get_player_info`)
+- [x] <strike>`get_slot`</strike> (will not be implemented -- returned part of `get_player_info`)
+- [ ] `set_slot`
+- [x] `force_player_slot`
+- [ ] <del>`lua2json`</del> (DCS-gRPC bridge preferred)
+- [ ] <del>`json2lua`</del> (DCS-gRPC bridge preferred)
+- [ ] <del>`dostring_in`</del> (DCS-gRPC bridge preferred)
+- [ ] <del>`log`</del> (client applications should use their own logging)
+- [ ] <del>`trace`</del> (client applications should use their own logging)
+
+#### GameGUI API
+  - [ ] `load_mission`
+  - [ ] `load_next_mission`
+
+
+### Timer Service
+- [x] `getTime`
+- [x] `getAbsTime`
+- [x] `getTime0`
+- [ ] `scheduleFunction` (TODO -- should this be implemented?)
```

Comment:
Transposing @rurounijones remark from initial draft PR.

> I would be against it. Clients can all the APIs when they need to,

Reply:
If the server is populated, I agree that this seems irrelevant.

However, in the use cases of servers with sparse density, the `scheduleFunction` could be a useful trigger to state "how much model time has really lapsed" when servers pause/resume as players connect/disconnect.

I would prefer to find an alternative solution to the pause/unpause scenario and tracking model/mission time.

Reply:
Since it is marked as a TODO we can deal with this later.